src/compiler/transformers/utilities.ts

```diff
@@ -255,4 +294,44 @@ namespace ts {
             return result;
         };
     }
+
+    /**
+     * Gets all property declarations with initializers on either the static or instance side of a class.
+     *
+     * @param node The class node.
+     * @param isStatic A value indicating whether to get properties from the static or instance side of the class.
+     */
+    export function getInitializedProperties(node: ClassExpression | ClassDeclaration, isStatic: boolean): ReadonlyArray<PropertyDeclaration> {
+        return filter(node.members, isStatic ? isStaticInitializedProperty : isInstanceInitializedProperty);
+    }
+
+    /**
+     * Gets a value indicating whether a class element is a static property declaration with an initializer.
+     *
+     * @param member The class element node.
+     */
+    export function isStaticInitializedProperty(member: ClassElement): member is PropertyDeclaration {
+        return isInitializedProperty(member, /*isStatic*/ true);
+    }

```

Comment:
it seems backwards to have `initializedProperty` take a boolean for whether it is static, then to have `isStaticInitializedProperty` just always pass `true`.

Two options:

- Just use `isInitializedProperty` and don't make a new function just to pass `true`
- Create `isInstanceInitializedProperty and rid of the second parameter of `isInitializedProperty`.

I lean toward the first option as it is less invasive and more consistent with the existing style in the codebase (a bunch of boolean flags). It's non-Metz but gets the job done.

Reply:
please ignore if the helpers were as-is before your changes and just moved